lib/automate/modules/reminder.py

```diff
@@ -38,19 +40,23 @@ def notify(self, os, body):
             # TODO: Add windows call here
             pass
 
-    def run(self, _to, when, body, _sender):
+    def run(self, text, sender):
+        to, when, body = self.nlp(text)
+        return self.execute_task(to, when, body, sender)
+
+    def execute_task(self, _to, when, body, _sender):
```

Comment:
perhaps remove sender as a parameter from execute_task if it isn't used??

Reply:
I would say it is up for the one responsible for this module. I did not feel comfortable refactoring the code itself as I've barely been working with it.